src/main/java/coresearch/cvurl/io/sse/SseEventSourceBuilder.java

```diff
@@ -0,0 +1,53 @@
+package coresearch.cvurl.io.sse;
+
+import coresearch.cvurl.io.mapper.GenericMapper;
+import coresearch.cvurl.io.mapper.MapperFactory;
+import coresearch.cvurl.io.request.CVurl;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Builder class that is used to build the {@link SseEventSource} with possibility to override the default reconnection time
+ * and {@link GenericMapper} implementation that is used to parse raw data in event.
+ */
+public class SseEventSourceBuilder {
```

Comment:
I'm a little bit confused by this class. How can it be used to create the SseEventSource?

Reply:
`SseEventSourceBuilder` is created when one uses the `CVurl#sse` method. Then to create a `SseEventSoruce` one should call the `SseEventSourceBuilder#build` method.

Reply:
Ah, sorry for that. For some reason, I saw a completely different class in the builder.